qs_cfn_lint_rules/SentenceCase.py

```diff
@@ -108,6 +108,7 @@ def match(self, cfn):
             return matches
         else:
             custom_dict = self.get_custom_dict()
+            sentence_case_exclude = cfn.template["Metadata"]["SentenceCaseExclude"]
```

Comment:
```suggestion
            sentence_case_exclude = cfn.template.get("Metadata", {}).get("SentenceCaseExclude", [])
```

This will prevent a `KeyError` if either `Metadata` or `SentenceCaseExclude` is missing.

Reply:
Great feedback. Thanks @andrew-glenn!